tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Maybe we can use extends with a siroc preset?

Reply:
(Given conversation below I think we've moved away from presets?)